source/includes/power_zones/_update.md

```diff
@@ -0,0 +1,59 @@
+## Update a Power Zone
+
+Requires the `power_zones_write` scope
+
+Updates a power zone for the authenticated user.
+
+```shell
+curl --header "Authorization: Bearer 414c6a44a5f8b918830b370db05" -X PUT
+  -d power_zone[ftp]=300 api.wahooligan.com/v1/power_zones/56519
```

Comment:
@JustinDunnWahoo missing https:// prefix on url .. other examples have it.. also, remove the bearer token and use placeholder